src/autogluon/bench/Dockerfile

```diff
@@ -1,11 +1,12 @@
 FROM 763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-training:1.13.1-gpu-py39-cu117-ubuntu20.04-ec2
+ARG AG_BENCH_VERSION=latest
 
 WORKDIR /app/
 COPY . /app/
 
 
 RUN pip install --upgrade pip \
-    && pip install -e .
+    && pip install autogluon.bench==$AG_BENCH_VERSION
```

Comment:
That's right, since the context of `Dockerfile` has been moved to `/src/autogluon/bench/` instead of the root of the repo as before. One workaround is to upload to `Test PyPI` for testing.